cdap-docs/developers-manual/source/building-blocks/metadata.rst

```diff
@@ -0,0 +1,44 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _metadata:
+
+========
+Metadata
+========
+
+.. include:: ../../../_common/_includes/include-v320-beta-metadata.rst
+
+Metadata |---| consisting of **properties** (a list of key-value pairs) or **tags** (a
+list of keys) |---| can be set for both data and applications.
+
+Using the CDAP :ref:`Metadata HTTP RESTful API <http-restful-api-metadata>` to set,
+retrieve, and delete the metadata annotations of applications, datasets, streams, and
+other elements in CDAP.
```

Comment:
should we call out programs here, since that's the only other supported entity not mentioned in this sentence.